environment_setup/install_clion.sh

```diff
@@ -44,10 +44,10 @@ else
 	mkdir -p /tmp/bazelbuild
 	tar xfz /tmp/bazelbuild.tar.gz -C /tmp/bazelbuild
 	cd /tmp/bazelbuild/intellij*
-	sudo bazel build //clwb:clwb_bazel_zip --define=ij_product=clion-${clion_major_version}
```

Comment:
This was fixed in a PR that was merged into #760 branch. @garethellis0 requested the change to be made there instead of the `bazel-and-no-ros` branch.

Reply:
Oh that explains why I didn't see it. Will remove. @garethellis0 will you manually port the change then?

Reply:
reverted the `sudo` change